### DISCUSSION TITLE
buttons don't work

### DISCUSSION THREAD

**Discussion by adnjoo** [aws/codewhisperer-command-line-discussions#117] [Known bugs] [answered]
description:
these purple buttons don't work

shell integrations

enable accessibility

![Screenshot 2024-03-19 at 12 14 19 PM](https://github.com/aws/codewhisperer-command-line-discussions/assets/22807629/967d14ad-24e5-4110-996e-d33281c5eafb)

repro:

macbook air m1, 2020, Sonoma 14.1.1

1. installed CW from https://docs.aws.amazon.com/codewhisperer/latest/userguide/command-line-getting-started-installing.html
2. ran codewhisperer
3. clicked install and enable but nothing happened

also had to follow the steps listed in #109 to get `cw` command to work

**Comment by bennygoldman** [comment]
Also having this issue

**Comment by grant0417** [comment]
Can you try running `cw doctor`? If you are not having any issues the components should already be installed.

**Reply by codejagaban** [reply]
@grant0417 throws the error: `zsh: command not found: cw`

**Reply by adnjoo** [reply]
I had to follow metal-spoon's steps in #109 to get my `cw doctor` to work as before it would throw the ^ above error

**Comment by codejagaban** [comment]
I'm having exactly the same issue here, 

In addition to the intergrate button not working the logout button is also not working, I'm not logged, but it's showing a log-out button:

<img width="972" alt="Screenshot 2024-03-19 at 10 00 00 pm" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/34829458/a16b4479-71f2-4c24-a68b-b6b366b31ea9">

**Reply by codejagaban** [reply]
By the way, I have a question: How do I login to codewhisperer CLI? There's no section or page that prompts a user to login

**Comment by grant0417** [comment, marked as answer]
There was a bug with the app that was causing the dashboard to not work as expected. Try restarting the app and you should be able to login and have it work.
